echo/server.go

```diff
@@ -69,10 +69,18 @@ import (
 	persesMiddleware "github.com/perses/common/echo/middleware"
 )
 
-var hidePort bool
+var (
+	hidePort bool
+	// https cert for server
+	cert string
+	// https key for server
+	key string
+)
 
 func init() {
 	flag.BoolVar(&hidePort, "web.hide-port", false, "If true, it won't be print on stdout the port listened to receive the HTTP request")
+	flag.StringVar(&cert, "web.tls-cert", "", "The cert to use for the HTTPS server")
```

Comment:
Thinking about how it will be used for example in k8s, I supposed the certificate will come from a secret / a file that needs to be reading.

I supposed this flag should contain the certificate itself not a path to a file. So would it make sense to add another flag to set the path to the certificate file if it exists ?

Reply:
yeah, I think it should just be a file path. I can rename the flags to explain that better.

Reply:
ah yeah, echo is expecting a files for the key and the certificate. Can you rename it please ? and change the description as well to reflect that ? Thank you

I would suggest `web.tls-cert-file`